building foxy branch fails

Hello, I'm trying to build foxy branch on ade, it gives the following output:

```
runtao@ade:~/pendulum_ws$ colcon build
Starting >>> pendulum2_msgs
Starting >>> pendulum_utils
Finished <<< pendulum_utils [2.72s]                                    
Finished <<< pendulum2_msgs [6.13s]                     
Starting >>> pendulum_controller
Starting >>> pendulum_driver
--- stderr: pendulum_driver                                             
CMake Error at CMakeLists.txt:59 (find_package):
  By not providing "Findapex_test_tools.cmake" in CMAKE_MODULE_PATH this
  project has asked CMake to find a package configuration file provided by
  "apex_test_tools", but CMake did not find one.

  Could not find a package configuration file provided by "apex_test_tools"
  with any of the following names:

    apex_test_toolsConfig.cmake
    apex_test_tools-config.cmake

  Add the installation prefix of "apex_test_tools" to CMAKE_PREFIX_PATH or
  set "apex_test_tools_DIR" to a directory containing one of the above files.
  If "apex_test_tools" provides a separate development package or SDK, be
  sure it has been installed.


---
Failed   <<< pendulum_driver [1.77s, exited with code 1]
Aborted  <<< pendulum_controller [1.78s]
                                
Summary: 2 packages finished [8.07s]
  1 package failed: pendulum_driver
  1 package aborted: pendulum_controller
  2 packages had stderr output: pendulum_controller pendulum_driver
  3 packages not processed
```

--- a/ade/docker-base/Dockerfile
+++ b/ade/docker-base/Dockerfile
@@ -86,6 +86,7 @@ RUN apt-get update \
   && mkdir -p /root/ros2_$ROS_VERSION/src \
   && wget https://raw.githubusercontent.com/ros2/ros2/$ROS_VERSION/ros2.repos \
   && vcs import src < ros2.repos \
+  && git clone -b $ROS_VERSION https://gitlab.com/ApexAI/apex_test_tools src/apex_test_tools \
   && rosdep init \
   && rosdep update \
   && rosdep install --from-paths src --ignore-src --rosdistro $ROS_VERSION -y --skip-keys "console_bridge fastcdr fastrtps rti-connext-dds-5.3.1 urdfdom_headers" \

